www/pages/partners/[slug].tsx

```diff
@@ -0,0 +1,183 @@
+import { GetStaticPaths, GetStaticProps } from 'next'
+import Link from 'next/link'
+import { createClient } from '@supabase/supabase-js'
+import { Partner } from '~/types/partners'
+import { NextSeo } from 'next-seo'
+import DefaultLayout from '~/components/Layouts/Default'
+import SectionContainer from '~/components/Layouts/SectionContainer'
+import { Typography, IconChevronLeft, Divider, IconExternalLink } from '@supabase/ui'
+import { Swiper, SwiperSlide } from 'swiper/react'
+import Error404 from '../404'
+
+const supabase = createClient(
+  'https://obuldanrptloktxcffvn.supabase.co',
+  'eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJyb2xlIjoiYW5vbiIsImlhdCI6MTYzNzY1NjAxNSwiZXhwIjoxOTUzMjMyMDE1fQ.0sfp_Njf7l4g-nOCF5a1TQE11rPqtz8Y10uctIetkBA'
```

Comment:
looks like this one is hardcoded! Should we move it to .env?

Reply:
We can, but since it's just anon credentials I just took the same approach as in https://github.com/supabase/supabase/blob/master/www/pages/auth/Auth.tsx#L37-L41